ursh/indico_ursh/client/index.js

```diff
@@ -18,40 +18,41 @@
 import {handleAxiosError, indicoAxios} from 'indico/utils/axios';
 
 
-$(() => {
-    async function _makeUrshRequest(originalURL, event) {
-        const urshEndpoint = '/ursh';
-        let response;
-        try {
-            response = await indicoAxios.post(urshEndpoint, {
-                original_url: originalURL,
-            });
-            const data = response.data;
-            if (data.success) {
-                $(event.target).copyURLTooltip(data.msg).show();
-            } else {
-                $(event.target).qtip({
-                    content: {
-                        text: $T.gettext(`URL shortening service is unavailable: ${data.msg}`),
-                    },
-                    hide: {
-                        event: 'mouseleave',
-                        fixed: true,
-                        delay: 700,
-                    },
-                    show: {
-                        event: false,
-                        ready: true,
-                    }
-                }).show();
+async function _makeUrshRequest(originalURL, event) {
+    const urshEndpoint = '/ursh';
+
+    let response;
+    try {
+        response = await indicoAxios.post(urshEndpoint, {
+            original_url: originalURL,
+        });
+    } catch (error) {
+        handleAxiosError(error);
+    }
+
+    const data = response.data;
+    if (data.success) {
+        $(event.target).copyURLTooltip(data.msg).show();
+    } else {
+        $(event.target).qtip({
+            content: {
+                text: $T.gettext(`URL shortening service is unavailable: ${data.msg}`),
+            },
+            hide: {
+                event: 'mouseleave',
+                fixed: true,
+                delay: 700,
+            },
+            show: {
+                event: false,
+                ready: true,
             }
-        } catch (error) {
-            handleAxiosError(error);
-        }
+        }).show();
     }
+}
+
 
-    $('.ursh-get').on('click', (event) => {
-        const originalURL = $(event.target).attr('data-original-url');
-        return _makeUrshRequest(originalURL, event);
-    });
+$(document).on('click', '.ursh-get', (event) => {
+    const originalURL = $(event.target).attr('data-original-url');
+    return _makeUrshRequest(originalURL, event);
```

Comment:
no need to return that. but you might want to `event.preventDefault()` to avoid getting a `#` in the url from clicking the link